### DISCUSSION TITLE
Random Value Help

### DISCUSSION THREAD

**Discussion by BWLGeorge** [dialogic-godot/dialogic#655] [Q&A]
Howdy everybody, I'm trying to set a sequence up in a timeline where there are a couple of different things that can happen and I want it to be randomly chosen from a list.  I tried doing this by setting the value to a random number and then creating a series of "If value is equal to x" commands for each number in the range, but the random value command seems to set the value to a number in the range randomly the first time it is used and then locks it to that number.  I also tried amending this by setting the value to 0 before then setting it to a random number.  Am I doing something wrong, or is there currently any way to do this?

Pic of the timeline for reference:
![Random value](https://user-images.githubusercontent.com/81444144/144085636-b5970298-591e-4b13-8e07-dace9efc75e5.PNG)

**Comment by Jowan-Spooner** [comment]
Do I understand correctly, that it always chooses the same number?

**Reply by BWLGeorge** [reply]
Yeah, it always chooses the same number.  If I repeat the command, it will then choose a different number, but then always choose that second number.

**Comment by Jowan-Spooner** [comment]
This might be because of godot. I think you need to write `randomize()` somewhere in a script (for example in your main scenes `_ready()`) so it uses a different seed everytime. Maybe dialogic should automatically do that.

**Reply by BWLGeorge** [reply]
Thank you for the tip!  I'll try setting it up that way
